src/execution_tree/primitives/define_variable.cpp

```diff
@@ -97,6 +97,63 @@ namespace phylanx { namespace execution_tree { namespace primitives
         p->store(hpx::launch::sync, std::move(val));
     }
 
+    ///////////////////////////////////////////////////////////////////
+
+    void define_variable::store_set_1d(
```

Comment:
Are these functions (on `define_variable`) ever called?

Reply:
@hkaiser I just tried it without the overloads and It worked fine without it. Will remove it in the next commit.